src/plugins/orangehrmLDAPAuthenticationPlugin/Service/LDAPService.php

```diff
@@ -34,6 +35,11 @@ class LDAPService
 
     protected ?AdapterInterface $adapter = null;
 
+    /**
+     * @var LDAPDao
+     */
+    private LDAPDao $LDAPDao;
```

Comment:
```suggestion
    private LDAPDao $ldapDao;
```
not matching with case in `getLDAPDao`

Reply:
done